scrapers/joymii.yml

```diff
@@ -0,0 +1,34 @@
+name: "joymii"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - joymii.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="info col-xs-12"]
+    scene:
+      Title: $info//h1[@class="title"]/text()
+      Details: $info//p/text()
```

Comment:
In scenes with multiple paragraphs in the description we need to concat them
`https://joymii.com/site/set-photo/code/20785844_clk308`
This should work
```yaml
      Details: 
        selector: $info//p/text()
        concat: "\n"
        replace:
          - regex: "\n"
            with: "\n"
```
the extra replace is needed (for now at least) as mentioned in another PR to protect the newlines added by concat